.github/actions/build-schemas/action.yaml

```diff
@@ -0,0 +1,33 @@
+name: Get schemas
+description: Gets new schemas from cluster
+runs:
+  using: composite
+  steps:
+    - name: Authenticate and set context
+      uses: redhat-actions/oc-login@v1
+      env:
+        OPENSHIFT_NAMESPACE: default
+      with:
+        openshift_server_url: ${{ inputs.OPENSHIFT_SERVER }}
+        openshift_token: ${{ inputs.OPENSHIFT_TOKEN }}
+        insecure_skip_tls_verify: true
+    - name: Update schemas
+      shell: bash
+      run: |
+        pip install openapi2jsonschema
+        cd $GITHUB_WORKSPACE
+        python build_schema.py -u $(oc whoami --show-server) -t $(oc whoami -t) -s STRICT
+        oc logout
```

Comment:
`oc logout` is not necessary here.

The only thing `oc logout` does is that it cleans up local kubeconfig. This config is gone anyways with the job completion.

Reply:
See https://github.com/openshift/oc/blob/master/pkg/cli/logout/logout.go for details :slightly_smiling_face: